src/components/form/skill/AddSkill.js

```diff
@@ -1,35 +1,173 @@
-import React from 'react';
-import { Close, Trash } from '~/assets/image';
+import * as Yup from 'yup';
+import * as _ from 'lodash';
+import PropTypes from 'prop-types';
+import { Formik, Form } from 'formik';
+import React, { useContext } from 'react';
+
+import { ADD } from '~/components/icons/icon';
 import Button from '~/components/button/Button';
+import * as storage from '~/storage/LocalStorage';
+import { FormContext } from '~/components/FormContext';
+import * as skillUtils from '~/utilities/objects/Skill';
 import InputText from '~/components/inputtext/InputText';
 import FormHeader from '~/components/formheader/FormHeader';
-import InputSelect from '~/components/inputselect/InputSelect';
 
-const AddContactInformation = () => {
+const AddSkill = ({ onClose, isEdit, values }) => {
+  const { data } = useContext(FormContext);
+
+  const skillsList = [];
+
+  const validateSkill = Yup.object().shape({
```

Comment:
shall we move this validation to /validation folder?

Reply:
this is no longer in use so removed.